### PR TITLE
[RFR] fix for access control tests

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
the test changes available menus but doesn't update menu object after change

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 510f6e11cae3b5a21744cafb169e3bce582ae5f5:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e3d987dadb5c1a9ae714f1c0ef9fafdcd93222e6:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e8db5bcf998f8986e438ddd9699341a570527be1:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -482,6 +488,7 @@ def _mk_role(name=None, vm_restriction=None, product_features=None):
 
 def _go_to(dest):
     """Create a thunk that navigates to the given destination"""
+    nav.initialize()
     return lambda: nav.go_to(dest)
```
Comment (psav) [inline cfme/tests/configure/test_access_control.py]: What we should do here is force_navigate, and not nav go_to

Comment (izapolsk) [inline cfme/tests/configure/test_access_control.py]: you're right. Actually, nav.initiallize + nav.go_to = force_navigate.

**Comment by dajoRH** [comment]
Lint report for commit e4aca93aeb39b354573f6c7ea695e58e6b320294:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 49188e4213fa5072499c931d9453276f1ae605cf:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cfe054feca40e26e5b9786d65f15f9829573d83c:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/fixtures/pytest_selenium.py [hunk 2]
```diff
@@ -112,25 +112,37 @@ def elements(o, **kwargs):
...
-        else:
-            return els
-    elif callable(o):
-        els = elements(o(), **kwargs)
-        if check_visibility:
-            return [e for e in els if is_displayed(e)]
+    if 'tries' in kwargs:
+        tries = kwargs['tries']
+    else:
+        tries = 3
```
Comment (psav) [inline cfme/fixtures/pytest_selenium.py]: This has the potential to slow things down by taking the triple the amount of time to time out on a block, what are we introducing it for? If it's a single specific place, we could just use a wait_for with is_displayed?

File: cfme/tests/configure/test_access_control.py [hunk 3]
```diff
@@ -104,7 +106,7 @@ def test_user_duplicate_name():
 @pytest.mark.tier(3)
 def test_username_required_error_validation():
     user = User(
-        name=None,
+        name="",
```
Comment (psav) [inline cfme/tests/configure/test_access_control.py]: Why empty string? and now None?

File: cfme/fixtures/pytest_selenium.py (hunk 2 above)
Comment (izapolsk) [inline cfme/fixtures/pytest_selenium.py]: I already removed this. I thought the issue was there but I already figured out exact reason.

**Comment by dajoRH** [comment]
Lint report for commit 0b6e6888070113681b8457f66d47d91fb948a1de:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py (hunk 3 above)
Comment (izapolsk) [inline cfme/tests/configure/test_access_control.py]: actually,  this change is not necessary but from my perspective empty "name" input field should be empty string not None.
 I can remove this change if you want

**Comment by dajoRH** [comment]
Lint report for commit 97b75e54a2cf35109a7c832510ddbd0e40eda2b0:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e3540e7e921346cc4fc1d82b9bd33eb2eb14011f:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**